src/validators/model.rs

```diff
@@ -35,6 +36,11 @@ impl BuildValidator for ModelValidator {
         // models ignore the parent config and always use the config from this model
         let config: Option<&PyDict> = schema.get_as("config")?;
 
+        let field_default_required = match config {
```

Comment:
```suggestion
        let model_full = match config {
```
Less confusing name.